### PR TITLE
Add 0.5.0 release information.

### PR DESCRIPTION
**Pull request by yaohaizh**
@andxu @jdneo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +22,16 @@ A lightweight extension to provide additional Java project explorer features. It
...
 
+
+## Settings
+
+| Setting Name | Description | Default Value |
+|---|---|---|
+| `java.dependency.showOutline` | Specify whether to show the outline in the dependency viewer. | `true` |
+| `java.dependency.syncWithFolderExplorer` | Specify whether to sync the folder with dependency viewer when browsering files  | `true` |
+| `java.dependency.autoRefresh` | Specify whether to automatically sync the change from editor to the dependency viewer | `true` |
+| `java.dependency.packagePresentation` | Specify how to display the package  Supported values are: `flat`, `hierarchical`| `flat` |
```
Comment (jdneo) [inline README.md]: Missing a `.` before `Supported ...`

File: README.md [hunk 2]
```diff
@@ -20,6 +22,16 @@ A lightweight extension to provide additional Java project explorer features. It
 - VS Code (version 1.28.0 or later)
 - [Language Support for Java by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java) (version 0.32.0 or later)
 
+
+## Settings
+
+| Setting Name | Description | Default Value |
+|---|---|---|
+| `java.dependency.showOutline` | Specify whether to show the outline in the dependency viewer. | `true` |
```
Comment (jdneo) [inline README.md]: Let's just make sure all the sentences are ended with `.`

**Comment by yaohaizh** [comment]
@jdneo Updated

**Comment by yaohaizh** [comment]
test this please

**Pull request merged**